specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-08-01-preview/managedClusters.json

```diff
@@ -0,0 +1,1463 @@
+      "description": "Managed cluster Access Profile.",
+      "x-ms-azure-resource": false
+    },
+    "AccessProfile": {
+      "type": "object",
+      "properties": {
+        "kubeConfig": {
+          "type": "string",
+          "format": "byte",
+          "description": "Base64-encoded Kubernetes configuration file."
+        }
+      },
+      "description": "Profile for enabling a user to access a managed cluster."
+    },
+    "ManagedClusterPoolUpgradeProfile": {
+      "properties": {
+        "kubernetesVersion": {
+          "type": "string",
+          "description": "Kubernetes version (major, minor, patch)."
+        },
+        "name": {
+          "type": "string",
+          "description": "Pool name."
+        },
+        "osType": {
+          "$ref": "#/definitions/OSType",
+          "enum": [
+            "Linux",
+            "Windows"
+          ],
+          "x-ms-enum": {
+            "name": "ContainerServiceOSTypes",
+            "modelAsString": true
+          },
+          "description": "OsType to be used to specify os type. Choose from Linux and Windows. Default to Linux."
+        },
+        "upgrades": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of orchestrator types and versions available for upgrade."
+        }
+      },
+      "required": [
+        "kubernetesVersion",
+        "osType"
+      ],
+      "description": "The list of available upgrade versions."
+    },
+    "ManagedClusterUpgradeProfileProperties": {
+      "properties": {
+        "controlPlaneProfile": {
+          "$ref": "#/definitions/ManagedClusterPoolUpgradeProfile",
+          "description": "The list of available upgrade versions for the control plane."
+        },
+        "agentPoolProfiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedClusterPoolUpgradeProfile"
+          },
+          "description": "The list of available upgrade versions for agent pools."
+        }
+      },
+      "required": [
+        "controlPlaneProfile",
+        "agentPoolProfiles"
+      ],
+      "description": "Control plane and agent pool upgrade profiles."
+    },
+    "ManagedClusterAADProfile": {
+      "properties": {
+        "clientAppID": {
+          "type": "string",
+          "description": "The client AAD application ID."
+        },
+        "serverAppID": {
+          "type": "string",
+          "description": "The server AAD application ID."
+        },
+        "serverAppSecret": {
+          "type": "string",
+          "description": "The server AAD application secret."
+        },
+        "tenantID": {
+          "type": "string",
+          "description": "The AAD tenant ID to use for authentication. If not specified, will use the tenant of the deployment subscription."
+        }
+      },
+      "required": [
+        "clientAppID",
+        "serverAppID",
+        "serverAppSecret"
```

Comment:
This was removed in a PR a month ago - I think you need to rebase to get the latest changes.